src/main/java/com/java_essential/Cherednichenko/classwork/lesson_2/Student.java

```diff
@@ -54,11 +56,11 @@ public static void main(String[] args) {
         }
         Course course1 = new Course();
         Course course2 = new Course();
-        course1.setStudents(students[1], 0);
-        course1.setStudents(students[0], 1);
-        course2.setStudents(students[1], 0);
-        course2.setStudents(students[2], 1);
-        course2.setStudents(students[3], 2);
+        course1.setStudents(students[1]);
```

Comment:
you set Student objects with the same ids to courses
```suggestion
        course1.setStudents(students[1]);
```

Reply:
can one student study in several courses?
or not ?